Add link validator and Linting (#21)
SovereignCloudStack/docs: community/contribute/adding-docs-guide.md

The documentation files have to be in markdown format and...

- comply [SCS licensing guidelines](https://github.com/SovereignCloudStack/docs/blob/main/community/github/dco-and-licenses.md)
- match our
  - [markdown file structure guideline](https://github.com/SovereignCloudStack/docs/blob/main/community/contribute/doc-files-structure-guide-md)
  - linting Rules
  - [styleguide](https://github.com/SovereignCloudStack/docs/blob/main/community/contribute/styleguide.md)

### Step 2 – Adding your repo to the docs.json

File a Pull Request within the [docs-page](https://github.com/SovereignCloudStack/docs-page) repository and add your repo to the docs.package.json:

> ```json5

Reviewer: As you dropped the comments from here to provide valid (plain) JSON, I think my change regarding JSON5 can be reverted:
```suggestion
```json
```

[
  {

>     repo: "demo-organisation/demo-repository",

Reviewer: I think having keys without `"` only works in JS (and JSON5, YAML...), not in JSON.
```suggestion
    "repo": "demo-organisation/demo-repository",
```

    source: "doc/*.md",
    target: "docs",
    label: "demo-repository-label",
  },
]
```

| key    | description                                                                                                       |
| ------ | ----------------------------------------------------------------------------------------------------------------- |

> | repo   | link to github organisation and repository                                                                        |

Reviewer: ```suggestion
| repo   | reference to github organisation and repository                                                                   |
```

> | source | directory which shall be copied. Optional to specify only copying markdown files                                  |

Reviewer: ```suggestion
| source | path to content to copy: Either glob matching individual markdown files OR the path of a single directory         |
```
